src/lib.rs

```diff
@@ -45,9 +56,36 @@ fn type_from_attrs(attrs: &[Attribute], attrname: &str) -> Option<String> {
     None
 }
 
+/// Defines the casing for the attributes long representation.
+#[derive(Copy, Clone, Debug, PartialEq)]
+enum CaseStyle {
+    Camel,
+    Kebab,
+    Mixed,
+    ShoutySnake,
+    Snake,
+    Verbatim,
+}
+
+impl CaseStyle {
+    fn from_string(name: &str) -> Self {
+        let normalized = name.to_camel_case().to_lowercase();
+        match normalized.as_ref() {
+            "camel" | "camelcase" => CaseStyle::Camel,
```

Comment:
I would prefer if there was only one option for each, and each option was the same as it's [`serde` counterpart](https://serde.rs/container-attrs.html), so: `camelCase`, `kebab-case`, `PascalCase` etc.

(I know this is a bit confusing as `heck` has a different idea of what they are called, particularly `PascalCase` => `CamelCase` and `CamelCase` => `mixedCase`)

Reply:
Done!